ralway env sample.txt

```diff
@@ -0,0 +1,12 @@
+RAILS_MAX_THREADS=3
+PORT=3000
+DATABASE_URL=
+LANG=es_US.UTF-8
+RACK_ENV=production
+RAILS_ENV=production
+RAILS_MASTER_KEY=
+PGDATABASE=
+PGHOST=
+PGPASSWORD=
+PGPORT=
+PGUSER=
```

Comment:
Just check that the name of the original file is the same as the one you are adding to it with the difference that at the end you add the word .example, example:

If your original file is called

`env.yml`

Your example file would be called:

`env.yml.example`

Also put an example of what the variables could have, for example in the PGHOST, you could put localhost, as an indication of what the developer using your source code can use as a base.